src/Roslyn/Core/Documentation/DoNotUseVerbatimCrefsAnalyzer.cs

```diff
@@ -37,6 +37,11 @@ public override ImmutableArray<DiagnosticDescriptor> SupportedDiagnostics
 
         protected void ProcessAttribute(SyntaxNodeAnalysisContext context, SyntaxTokenList textTokens)
         {
+            if (!textTokens.Any())
+            {
+                return;
+            }
+
             var token = textTokens.First();
 
             if (token.Span.Length >= 2)
```

Comment:
This is unrelated to the fix, but I think these hardcoded numbers deserve to have a comment :)